strax/plugin.py

```diff
@@ -886,8 +886,19 @@ def inline_plugins(cls, components, start_from, log):
         p.sub_savers = sub_savers
         p.start_from = start_from
         if p.multi_output:
-            p.dtype = {d: p.sub_plugins[d].dtype_for(d)
-                       for d in outputs_to_send}
+            p.dtype = {}
+            for d in outputs_to_send:
+                if d in p.sub_plugins:
+                    p.dtype[d] = p.sub_plugins[d].dtype_for(d)
+                else:
+                    log.debug(f'Finding plugin that provides {d}')
+                    # Need to do some more work to get the plugin that
+                    # provides this data-type.
+                    for sp in p.sub_plugins.values():
+                        if d in sp.provides:
+                            log.debug(f'{sp} provides {d}')
+                            p.dtype[d] = sp.dtype_for(d)
+                            break
```

Comment:
Have you checked what happens if you cannot find `d` in `sp.provides.values()`? Is the traceback already enough or shall we just add a small raise condition after the for-loop?

Reply:
If that happens, there is some deeper bug in strax because it asks you to produce a datatype that is not provided by any of the multi-process plugins, that is troublesome.